src/System.ComponentModel.Annotations/tests/DataTypeAttributeTests.cs

```diff
@@ -68,6 +68,16 @@ public static void GetDataTypeName_and_IsValid_on_empty_custom_DataTypeAttribute
         }
 
         [Fact]
+        public static void GetDataTypeName_and_IsValid_on_whitespace_custom_DataTypeAttribute_throws_exception()
+        {
+            var attribute = new DataTypeAttribute(new String(' ', 5)); // Create a string with five whitespaces
```

Comment:
You are right. I wanted to take the test on a case more complicated. Sorry. I will modify the test.